lib/src/test/java/com/auth0/jwt/JWTDecoderTest.java

```diff
@@ -48,7 +49,7 @@
     @Test
     public void getSubject() throws Exception {
         String token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWV9.TJVA95OrM7E2cBab30RMHrHDcEfxjoYZgeFONFh7HgQ";
-        JWT jwt = JWT.require(Algorithm.HMAC256("secret")).withNonStandardClaim("admin", true).withNonStandardClaim("name", "John Doe").build();
+        JWT jwt = JWT.require(Algorithm.HMAC256("secret")).withNonStandardClaim("admin", true).withNonStandardClaim(Claims.NAME, "John Doe").build();
```

Comment:
Don't use constants on the tests, or define them here in this class instead of using that `Claims` class. You could change the `Claims` values and the test would keep passing when it shouldn't

Reply:
i dont get how it would keep passing.  if you change a value, it should fail.